python/Ganga/new_tests/Unit/TestGangaObject.py

```diff
@@ -296,9 +296,9 @@ def write_read(thread_number):
                     child_num = rand.randint(0, 1000)
                     o.b.a = child_num
                     time.sleep(rand.uniform(0, 1E-6))
-                    self.assertEqual(o.a, num)
```

Comment:
Just for completeness, which are we favoring in the future, ```assertEqual``` and equivalent of plain old ```==```?